pulp_gem/tests/functional/api/test_pull_through.py

```diff
@@ -60,18 +60,22 @@ def test_pull_through_install(
     remote2 = gem_remote_factory()
     distribution2 = gem_distribution_factory(remote=remote2.pulp_href)
 
-    for dis, gem in ((distribution, "a"), (distribution2, "beryl")):
-        cmd = ["gem", "i", "--remote", "--clear-sources", "-s", dis.base_url, gem, "-v", "0.1.0"]
+    for dis, gem, version in (
+        (distribution, "a", "0.1.0"),
+        (distribution2, "beryl", "0.1.0"),
+        (distribution, "ruby-hmac", "0.4.0"),
```

Comment:
Can you verify that the requirements are correctly imported in the pulp database?

Reply:
Yes I can!

Reply:
{'hoe': '>= 2.5.0', 'gemcutter': '>= 0.2.1', 'rubyforge': '>= 2.0.3'}

Reply:
I added dependency assertions at the end of the test.